src/components/RnR/RnR_container.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class RnR extends React.Component {
+  constructor() {
```

Comment:
If you're using props in a class component you'd need to pass props to the constructor and super.